Cargo.toml

```diff
@@ -8,3 +8,6 @@ members = [
 [profile.release]
 lto = true
 codegen-units = 1
+
+[patch.crates-io]
+openssl-src = { git = "https://github.com/blu3beri/openssl-src-rs", branch = "release/111" }
```

Comment:
Maybe add a comment here why it's patched?

Reply:
> > Where does it mention they only accept security PRs? Not sure what to do if we can't get it in. This hasn't been solved for indy-credx yet a while ago?
> 
> Here for example: [alexcrichton/openssl-src-rs#130 (comment)](https://github.com/alexcrichton/openssl-src-rs/pull/130#issuecomment-1069625131)
> 
> I don't think it was fixed for credx, I vaguely remember the issue from before. Would be nice if we can just have a patch file in this repo with the fix. Can check for that later on.

We may be able to say it IS pressing as otherwise we'll have forks that don't necesarily improve security 🤷

Reply:
Yeah I will create a PR with it. If it is merged I will remove the fix but leave this in for now.